style.css

```diff
@@ -0,0 +1,416 @@
+	background: #FFFF00;
+	position: absolute;
+	left: 230px;
+	bottom: 15px;
+}
+
+.call-advertising-number-why:hover .call-advertising-because {
+	display: block;
+}
+
+.product-image-container {
+	width: 60%;
+	float: left;
+}
+
+.product-details-container {
+	width: 40%;
+	float: right;
+}
+
+.product-redirect {
+	padding-top: 10px;
+	font-size: 14px;
+	margin-bottom: 40px;
+}
+
+.product-category-link {
+	color: #68899a;
+	text-decoration: none;
+}
+
+.product-price {
+	font-size: 24px;
+	color: #68899a;
+	margin-bottom: 25px;
+}
+
+.product-name {
+	font-size: 24px;
+	margin-top: 0;
+	margin-bottom: 18px;
+}
+
+.product-rating-review {   /* todo: need style for this block, i get back to this point later */
+	padding-bottom: 20px;
+}
+
+.product-additional-info-availability {
+	font-size: 14px;
+}
+
+.product-additional-info-property-value {
+	color: #68899a;
+}
+
+.product-tag {
+	color: #68899a;   /* zzhanna4796 */
+	text-decoration: none;
+}
+
+.product-additional-info-availability {
+	margin-top: 10px;
+	font-weight: bold;
+}
+
+.product-description {
+	margin: 27px 0 40px;
+	font-size: 14px;
+	color: #555555;
+}
+
+.product-description li {
+	list-style-type: disc;
+	list-style-position: inside;
+	color: #555555;
+}
+
+.product-order-form {
+
+}
+
+.color-selector {
+	font-size: 14px;
+	font-weight: bold;
+}
+
+.product-order-form-label {
+	font-size: 14px;
+	font-weight: bold;	
+}
+
+.quantity-select-wrapper-label {
+	font-size: 14px;
+	font-weight: bold;	
+}
+
+.product-order-form-select {
+	width: 165px;
+	height: 35px;
+	color: #555555;
+	padding-left: 15px;
+}
+
+.product-order-quantity {
+	width: 70px;
+	height: 35px;
+	color: #555555;
+	padding-left: 15px;
+	margin-bottom: 40px;
+}
+
+.main-image-wrapper {
+	margin-bottom: 20px;
+}
+
+.product-order-form-reset-button {
+	font-size: 14px;
+	font-weight: bold;
+	border: 0;
+	background-color: #ffffff;
+	text-decoration: underline;
+	text-transform: uppercase;
+	margin-bottom: 40px;
+	outline: 0;
+}
+
+.thumbnails {
+	width: 100%;
+	display: block;
+	margin-bottom: 100px;
+}
+
+.thumbnails > li {
+	display: inline-block;
+}
+
+.thumbnails li {
+	padding-right: 19px;
+}
+
+.buttons-row {
+	float: right;
+	display: block;
+	width: 30%;
+}
+
+.buttons-row  > li {
+	display: inline-block;
+}
+
+.add-to-cart {
+	font-size: 13px;
+	text-transform: uppercase;
+	text-decoration: none;
+	background-color: #080808;
+	padding: 5px 60px;
+	text-align: center;
+}
+
+.add-to-wishlist {
+	font-size: 13px;
+	text-transform: uppercase;
+	text-decoration: none;
+	padding: 5px 60px;
+	border: 1px solid #e1e1e1;
+	color: #555555;
+	text-align: center;
+}
+
+.buttons-row:after {
+	content: "";	
+	display: block;
+	height: 0;
+	clear: both;
+}
+
+.social-block {
+	height: 50px;
+	border-top: 1px solid #e1e1e1;
+	border-bottom: 1px solid #e1e1e1;
+}
+
+.social-block-button-links {
+	
+}
+
+.footer {
+	border-top: 1px solid #e1e1e1;
+	height: 70px;
+}
+
+.copyright {
+	width: 50%;
+	margin: 25px auto;
```

Comment:
don't use position-related rules to the deepest child - use positioning as higher as possible (use padding for .footer or .wrapper)